rick_morty_app/pubspec.yaml

```
@@ -0,0 +1,119 @@
+name: rick_morty_app
+description: "A new Flutter project."
+# The following line prevents the package from being accidentally published to
+# pub.dev using `flutter pub publish`. This is preferred for private packages.
+publish_to: 'none' # Remove this line if you wish to publish to pub.dev
```

YAMLlint (1.37.1)

[warning] too few spaces before comment: expected 2

(comments)